feat : add basic structure for module screenNvideo

Chirag9528 opened the pull request.

alonot committed:
Contributing.md reviewer changes

andhkr committed:
"Lossy Compression of Image Block and Returning the String of YbCbCr component of block as  first part of String reperesent Y then Cb and then Cr"

BhupathiVarun committed:
video capture

LOGANATH07 committed:
Initial Networking base for prototyping...

sandeepkumar009 committed:
feat: add ScreenCapture utility for cross-platform screen capture

Devansh-Kesan committed:
Added ImageStitcher module

LOGANATH07 committed:
Initial prototype networking base...

LOGANATH07 committed:
Initial prototype networking base...

Chirag9528 committed:
Added Decoder method and restructed Code

Written a Codec Interface and a JpegCodec class which  implements the
Codec interface.
Deleted Compressor.java and YCbCr.java files.
Written Decoder method of JpegCodec Class.

aman112201041 committed:
first implemention od PacketGenerator with CompressedPatch, ICompressor and IHasher

aman112201041 committed:
added Hasher

alonot committed:
Merge pull request #6 from alonot/Devansh/feature/ImageStitcher

Added ImageStitcher module

alonot committed:
Merge pull request #3 from alonot/anup/feature/compressor-lossy

Lossy Compression Added

alonot committed:
Merge pull request #8 from alonot/aman/feature/PacketGenerator

Aman/feature/packet generator

alonot committed:
Merge pull request #7 from alonot/chirag/feature/decompressor

Codec Implementation

alonot committed:
Merge pull request #9 from alonot/video_capture

 video capture

alonot committed:
merged main

alonot committed:
Refactored the code and fixed accessibility issues

sandeepkumar009 committed:
Created ICapture interface and updated ScreenCapture to implement it

alonot committed:
Merge pull request #5 from alonot/Sandeep_Kumar/feature/ScreenCapture

feat: add ScreenCapture utility for cross-platform screen capture

alonot committed:
Merge branch 'module/ScreenNVideo' of github.com:alonot/Java-Core-Communicator-SWE into priyanshu/feature/media_manager

alonot committed:
MediaManager basic interface with networking and RPC

LOGANATH07 committed:
Added getServer function

hemant030406 committed:
implement ClientServer's sendTo function

909Ahmed committed:
PacketParser

Implements the packet parser for the proposed packet design
- Contains method for packet creation given attributes
- Contains get methods to retrieve attributes given packet

Files Changed:
PacketParser.java

Testing:
None

hemant030406 committed:
implement client server cluster, author: vishal

loganvegeta committed:
Merge pull request #1 from loganvegeta/hemant/implement-ClientServer-Sendto-function

Implement ClientServer's sendTo function

loganvegeta committed:
Merge pull request #2 from loganvegeta/asad/packetParser

PacketParser

loganvegeta committed:
Merge pull request #3 from loganvegeta/vishal/implement-cluster

Implement client server cluster, author: vishal

Vishwaa-Idiosyncrat committed:
Queue Manager with Tests created and added

loganvegeta committed:
Merge pull request #4 from loganvegeta/queueManager

Queue Manager with Tests created and added

udithdoddapaneni committed:
added recieveFrom function

loganvegeta committed:
Merge pull request #5 from loganvegeta/udith-networking

added recieveFrom function

LOGANATH07 committed:
Added Checkstyle for topology and added a test

loganvegeta committed:
Merge pull request #6 from loganvegeta/logan/getServer

Logan/get server

loganvegeta committed:
Merge pull request #7 from loganvegeta/main

Main

Vishwaa-Idiosyncrat committed:
Add files via upload

udithdoddapaneni committed:
Receive from function

loganvegeta committed:
Merge pull request #8 from loganvegeta/udith-networking-2

Receive from function

Vishwaa-Idiosyncrat committed:
Update PriorityQueue.java

Shubham04567 committed:
add User in topology

alonot committed:
MediaManager server side code skeleton

Chirag9528 committed:
Merge pull request #1 from alonot/priyanshu/feature/media_manager

Media Manager added

Chirag9528 committed:
restructured the module

alonot committed:
Merge pull request #10 from alonot/chirag/fix/project-structure

restructured the module

dhruvadeep commented on java/module-screen-video/src/main/java/com/swe/ScreenNVideo/PatchGenerator/CompressedPatch.java:
```diff
@@ -1,4 +1,4 @@
-package com.Comm_Uni_Cator.ScreenNVideo.PatchGenerator;
+package com.swe.ScreenNVideo.PatchGenerator;
 
```
Just a small information, could you add both the getters and setters here 
rather than using say public CompressedPatch(...)
use things like this.x = this.setX(...)

dhruvadeep commented on java/module-screen-video/src/main/java/com/swe/ScreenNVideo/PatchGenerator/Stitchable.java:
```diff
@@ -1,4 +1,4 @@
-package com.Comm_Uni_Cator.ScreenNVideo.PatchGenerator;
+package com.swe.ScreenNVideo.PatchGenerator;
 
 // Interface
```
be more explicit what this function/class is, write proper comment for this

dhruvadeep reviewed (commented):
Small info added, add proper comments so that we can understand the internals properly

loganvegeta committed:
Merge pull request #9 from loganvegeta/shubham/add-user-in-topology

add User in topology

loganvegeta committed:
Delete Networking/target directory

dhruvadeep commented on java/module-screen-video/src/main/java/com/swe/ScreenNVideo/Capture/VideoCapture.java:
```diff
@@ -0,0 +1,133 @@
+    // Listener for frame capture events
+    private FrameCaptureListener listener;
+
+    // Default capture settings
```
Define the values in some config file since these files needs to be changed for testing purposes.
Follow discord guide to check it

dhruvadeep commented on java/module-screen-video/src/main/java/com/swe/ScreenNVideo/Capture/VideoCapture.java:
```diff
@@ -0,0 +1,133 @@
+        try {
+            // Initialize Robot for screen capture
+            this.robot = new Robot();
+            System.out.println("VideoCapture initialized with screen capture");
```
Implement a logger rather than using println statements, this can be beneficial for testing and error testing also,

enchanteddev replied:
although better, lets not block this PR for logging just yet, we will move to logging sometime later.

dhruvadeep commented on java/module-screen-video/src/main/java/com/swe/ScreenNVideo/Capture/VideoCapture.java:
```diff
@@ -0,0 +1,133 @@
+            System.out.println("Capture area: " + captureArea.width + "x" + captureArea.height +
+                    " at (" + captureLocation.x + "," + captureLocation.y + ")");
+        } catch (AWTException e) {
+            System.err.println("Error initializing Robot: " + e.getMessage());
```
Same as above

dhruvadeep commented on java/module-screen-video/src/main/java/com/swe/ScreenNVideo/Capture/VideoCapture.java:
```diff
@@ -0,0 +1,133 @@
+    public void setCaptureArea(int x, int y, int width, int height) {
+        this.captureLocation = new Point(x, y);
+        this.captureArea = new Dimension(width, height);
+        System.out.println("Capture area updated: " + width + "x" + height +
```
.

dhruvadeep commented on java/module-screen-video/src/main/java/com/swe/ScreenNVideo/ConstantProvider.java:
```diff
@@ -0,0 +1,10 @@
+package com.swe.ScreenNVideo;
+
```
Use a config yaml or xml file for these values, its easier to control later on for client side also

enchanteddev replied:
we don't need to control these values from the user's end, if anything, we have to ensure it stays the same post compilation.

I say, keep it as it is, but make the formatting consistent on the last line

dhruvadeep commented on java/module-screen-video/src/main/java/com/swe/ScreenNVideo/ConstantProvider.java:
Change these values to some other file

dhruvadeep reviewed (changes requested):
Files need to be addressed

alonot committed:
pom.xml updates according to new project structure

alonot committed:
checkstyle check added

alonot committed:
Style check on push and PRs both

alonot committed:
step for styecheck.yml

LOGANATH07 committed:
Modified camelCase for variables and passed checkstyles

Devansh-Kesan committed:
solved Checkstyle errors from ImageStitcher

alonot committed:
Network Serializer and code refractor

alonot committed:
refractor : short[][][] to int[][]

alonot committed:
packet sending to other clients(viewers)

Chirag9528 committed:
Add BilinearScaler implementing ImageScaler interface

Added BilinearScaler class which implements ImageScaler interface.

Chirag9528 committed:
Refactor JpegCodec

replaced BufferedImage with int[][] RGB representation and changed
encoder output from string to array of bytes

Devansh-Kesan committed:
Updated and refactored Patch

Devansh-Kesan committed:
Updated and refactored Stitchable

Devansh-Kesan committed:
Updated and refactored ImageStitcher

alonot committed:
Merge pull request #11 from alonot/Devansh/checkstyle/ImageStitcher

solved Checkstyle errors from ImageStitcher

alonot committed:
Merge branch 'module/ScreenNVideo' into chirag_priyanshu/feat/integration-n-refractor

# Conflicts:
#	java/module-screen-video/src/main/java/com/swe/ScreenNVideo/PatchGenerator/ImageStitcher.java

alonot committed:
Feed mixing and Synchronizer implementation

enchanteddev commented on java/module-screen-video/src/main/java/com/swe/ScreenNVideo/PatchGenerator/Stitchable.java:
```diff
@@ -0,0 +1,6 @@
+package com.swe.ScreenNVideo.PatchGenerator;
+
+// Interface
+public interface Stitchable {
```
IStitchable, not Stitchable.

sandeepkumar009 committed:
fixed Checkstyle errors from Capture

alonot committed:
Integration Test skeleton.

Current Status : Not working

alonot committed:
Merge pull request #12 from alonot/Sandeep/fix/Checkstyle

fixed Checkstyle errors from Capture

Vishwaa-Idiosyncrat committed:
Fix priority queue logic and update related tests

alonot committed:
Testing with encoder and decoder...

> Status: Purple image received on other end

andhkr committed:
JPEG Codec out of bounds fix

andhkr committed:
Basic screensharing working with IntegrationTest.

Chirag9528 committed:
fix: Fixed Checkstyle warnings in Codec subdirectory

Restructed code and fixed checkstyle warnings in the following files:
- BilinearScaler.java
- Codec.java
- ImageScaler.java
- JpegCodec.java

Chirag9528 committed:
Merge branch 'module/ScreenNVideo' into chirag_priyanshu/feat/integration-n-refractor

> Refactored Code to fix some checkstyle errors

aman112201041 committed:
checkstyle errors removed from CompressedPatch, Hasher, IHasher and PacketGenerator

alonot committed:
Merge pull request #13 from alonot/aman/checkstyle_fix

checkstyle errors removed from CompressedPatch, Hasher, IHasher and P…

alonot committed:
Merge branch 'module/ScreenNVideo' into chirag_priyanshu/feat/integration-n-refractor

# Conflicts:
#	java/module-screen-video/src/main/java/com/swe/ScreenNVideo/Capture/ScreenCapture.java
#	java/module-screen-video/src/main/java/com/swe/ScreenNVideo/Capture/VideoCapture.java
#	java/module-screen-video/src/main/java/com/swe/ScreenNVideo/PatchGenerator/CompressedPatch.java
#	java/module-screen-video/src/main/java/com/swe/ScreenNVideo/PatchGenerator/Hasher.java
#	java/module-screen-video/src/main/java/com/swe/ScreenNVideo/PatchGenerator/PacketGenerator.java

VUPPALA-HEMANTH committed:
image captured by webcam

VUPPALA-HEMANTH committed:
all other files like *.class from capture folder and also jpg image within java

909Ahmed committed:
Tests added for the Packet Parser and also extending the packet parser to support the connectionType and BroadCast

loganvegeta committed:
Merge pull request #11 from loganvegeta/asad/packetParserTests

Tests added for the Packet Parser and also extending the packet parse…

loganvegeta committed:
Merge pull request #12 from loganvegeta/main

Main

Vishwaa-Idiosyncrat committed:
updated Priority Queue

LOGANATH07 committed:
Getnetwork function in topology and new structure for storing data

alonot committed:
Fixed glitches in video on client UI

909Ahmed committed:
Switch the packetParser to singleTon and added mainServer to clientServer

alonot committed:
Check styles errors fixs for /java/ScreenNVideo

LOGANATH07 committed:
Some modifications for the send and receive functions

909Ahmed committed:
changes in network packet structure

loganvegeta committed:
Merge pull request #13 from loganvegeta/asad/packetParserUpdate0

changes in network packet structure

909Ahmed committed:
added the MessageId, ChunkNum, chunkLength

loganvegeta committed:
Merge pull request #14 from loganvegeta/asad/PacketParser1

added the MessageId, ChunkNum, chunkLength

Vishwaa-Idiosyncrat committed:
Merge remote-tracking branch 'origin/module/networking' into vishwaa/queuemanager

udithdoddapaneni committed:
chunkManager added with Tests

Vishwaa-Idiosyncrat committed:
Updated Priority Queue and Created test for parallel access for the next packet and add packet

loganvegeta committed:
Merge pull request #15 from loganvegeta/vishwaa/queuemanager

Vishwaa/queuemanager

udithdoddapaneni committed:
Fixed bugs in chunk manager

udithdoddapaneni committed:
The chunkLength was wrongly set in some cases

ElectroZoid committed:
Update ClientServer with both mine and hemants code

hemant030406 committed:
Merge branch 'vishal/clientserver' of github.com:loganvegeta/Java-Core-Communicator-SWE into vishal/clientserver

ElectroZoid committed:
Update ClientServer more precise

hemant030406 committed:
Merge branch 'vishal/clientserver' of github.com:loganvegeta/Java-Core-Communicator-SWE into vishal/clientserver

hemant030406 committed:
add javadoc comments to align with checkstyle

loganvegeta committed:
Merge pull request #16 from loganvegeta/vishal/clientserver

Vishal/clientserver

LOGANATH07 committed:
Simple client server model

LOGANATH07 committed:
Some minor changes

LOGANATH07 committed:
client updated

LOGANATH07 committed:
Client Server model testing

loganvegeta committed:
Merge pull request #13 from loganvegeta/module/networking

Basic client server model

neverthesameagain committed:
Merge pull request #14 from Comm-Uni-Cator-SWE-2025/module/networking

Module/networking basic Client Server model

alonot committed:
Integrated Socketry

udithdoddapaneni committed:
Grouping and Merging chunks methods added

alonot committed:
Merge pull request #15 from alonot/chirag_priyanshu/feat/integration-n-refractor

Chirag priyanshu/feat/integration n refractor

alonot committed:
Merge branch 'main' of github.com:Comm-Uni-Cator-SWE-2025/Java-Core-Communicator-SWE into priyanshu/integration/networking

alonot committed:
Merge branch 'module/ScreenNVideo' of github.com:alonot/Java-Core-Communicator-SWE into priyanshu/integration/networking

udithdoddapaneni committed:
fixed bugs in GroupChunks and MergeChunks and added tests

alonot committed:
networking module added

loganvegeta committed:
Update README.md

loganvegeta committed:
Update README.md  AB#64

LOGANATH07 committed:
TCP Communicator class and tests completed AB#71

LOGANATH07 committed:
Added base class files for P2P Cluster AB#72

Vishwaa-Idiosyncrat committed:
Updated the new logic for the nextPacket and added new tests

Vishwaa-Idiosyncrat committed:
Merge remote-tracking branch 'origin/module/networking' into vishwaa/queuemanager

Vishwaa-Idiosyncrat committed:
Updated the new logic for the nextPacket function and added new tests AB#73

LOGANATH07 committed:
Some changes and test for mainserver AB#74

Vishwaa-Idiosyncrat committed:
Updated the new logic for the nextPacket function and added new tests AB#73

Vishwaa-Idiosyncrat committed:
Merge remote-tracking branch 'origin/module/networking' into vishwaa/queuemanager

loganvegeta committed:
Merge pull request #20 from loganvegeta/vishwaa/queuemanager

Vishwaa/queuemanager

LOGANATH07 committed:
Added tests and unit tests for SimpleNetworking module AB#84

LOGANATH07 committed:
Merge branch 'module/networking' of github.com:loganvegeta/Java-Core-Communicator-SWE into module/networking

LOGANATH07 committed:
Small change in test AB#84

LOGANATH07 committed:
Changes in main server

LOGANATH07 committed:
Small changes in checkstyle

Chirag9528 committed:
fix: Fix checkstyle errors

LOGANATH07 committed:
Added serializer in networking class AB#94

LOGANATH07 committed:
Changes in CallSubscriber function

alonot committed:
Code refractor:
- integration with networking
- ip is sent to UI along with the image

alonot committed:
fix: Different users images mixing with each other

udithdoddapaneni committed:
only one checkstyle issue: related to number of parameters

alonot committed:
Merge pull request #16 from alonot/chirag/fix/checkstyle

fix: Fix checkstyle errors

alonot committed:
Merge branch 'module/ScreenNVideo' of github.com:alonot/Java-Core-Communicator-SWE into priyanshu/integration/networking

neverthesameagain committed:
Merge pull request #18 from loganvegeta/module/networking

Unit  tests for simpleNetworking and fixed some bugs

alonot committed:
Check style only checks for module/screenNVideo

loganvegeta committed:
Merge pull request #21 from loganvegeta/udith/networking/chunkManager

Udith/networking/chunk manager

LOGANATH07 committed:
Changes to Simple Networking model

LOGANATH07 committed:
Some directory changes

loganvegeta committed:
Merge pull request #21 from loganvegeta/module/networking

SimpleNetworking updated

alonot committed:
UI update moved to separate thread

909Ahmed committed:
Added code for the p2pserver and p2pcluster and added timer.java

909Ahmed committed:
updated the packetparser

alonot committed:
Merge branch 'module/networking' of github.com:Comm-Uni-Cator-SWE-2025/Java-Core-Communicator-SWE into priyanshu/integration/networking

# Conflicts:
#	java/module-networking/pom.xml
#	java/module-networking/src/main/java/com/swe/networking/SimpleNetworking/AbstractController.java
#	java/module-networking/src/main/java/com/swe/networking/SimpleNetworking/AbstractNetworking.java
#	java/module-networking/src/main/java/com/swe/networking/SimpleNetworking/Client.java
#	java/module-networking/src/main/java/com/swe/networking/SimpleNetworking/IUser.java
#	java/module-networking/src/main/java/com/swe/networking/SimpleNetworking/Server.java
#	java/module-networking/src/main/java/com/swe/networking/SimpleNetworking/SimpleNetworking.java
#	java/module-networking/src/main/java/com/swe/networking/TCPCommunicator.java
#	java/module-networking/src/main/java/com/swe/networking/Topology.java
#	java/module-networking/src/test/java/com/swe/networking/PriorityQueueTest.java
#	java/module-networking/src/test/java/com/swe/networking/TopologyTest.java

loganvegeta committed:
Merge pull request #23 from loganvegeta/asad/P2PServer

Added code for the p2pserver and p2pcluster and added timer.java

loganvegeta committed:
Merge pull request #24 from loganvegeta/asad/checkStyle_fixes

updated the packetparser

LOGANATH07 committed:
Checkstyle changes

909Ahmed committed:
added tests for p2pserver and fixed some bugs in serializers

loganvegeta committed:
Merge pull request #25 from loganvegeta/asad/P2PServerTest

added tests for p2pserver and fixed some bugs in serializers

Vishwaa-Idiosyncrat committed:
Updated the new logic that was changed for packet structure and updated the number of priorities AB#73

Vishwaa-Idiosyncrat committed:
Merge remote-tracking branch 'origin/module/networking' into vishwaa/queuemanager

loganvegeta committed:
Merge pull request #26 from loganvegeta/vishwaa/queuemanager

Vishwaa/queuemanager

LOGANATH07 committed:
Fixed some bugs and checkstyles

LOGANATH07 committed:
Checkstyle for simplenetworking/Packetparser

LOGANATH07 committed:
Topology changes

Shubham04567 committed:
p2p client implemetation AB#98

loganvegeta committed:
Merge pull request #27 from loganvegeta/p2pclient-implementation

p2p client implemetation AB#98

LOGANATH07 committed:
Checkstyle fixes

LOGANATH07 committed:
Timer added to main server AB#99

hemant030406 committed:
implement coalescing send

hemant030406 committed:
implement coalescing send #100

hemant030406 committed:
Merge branch 'hemant/feat/coalesce-send' of github.com:loganvegeta/Java-Core-Communicator-SWE into hemant/feat/coalesce-send

hemant030406 committed:
implement coalescing send #100

loganvegeta committed:
Merge pull request #28 from loganvegeta/hemant/feat/coalesce-send

Coalesce Send

LOGANATH07 committed:
Reoved priority queue temporarily

LOGANATH07 committed:
Fixed remve client bug in topology

LOGANATH07 committed:
Fixed remove client bug in topology

Vishwaa-Idiosyncrat committed:
Updated the logic for Queue Manager #AB73

Vishwaa-Idiosyncrat committed:
Resolve merge conflict: keep fixed PriorityQueue files #AB73

Vishwaa-Idiosyncrat committed:
Resolve merge conflict: keep fixed PriorityQueue files #AB73

loganvegeta committed:
Merge pull request #29 from loganvegeta/vishwaa/queuemanager

Vishwaa/queuemanager

LOGANATH07 committed:
timer implemented for main serverand tested AB#99

LOGANATH07 committed:
Checkstyle fixes for the changes made AB#99

LOGANATH07 committed:
splitting packets on receiving

909Ahmed committed:
tested the servers init

909Ahmed committed:
updates in parser

loganvegeta committed:
Merge pull request #31 from loganvegeta/asad/ServerTests

tested the servers init

loganvegeta committed:
Merge pull request #32 from loganvegeta/asad/packetParserUpdate

updates in parser

LOGANATH07 committed:
fixed buggy split packets and some tests AB#104

LOGANATH07 committed:
Fixed split packets tests

BhupathiVarun committed:
added dependencies in pom.xml with version change to 25 for java

BhupathiVarun committed:
added modified VideoCapture.java

BhupathiVarun committed:
added TestVideo.java

Vishwaa-Idiosyncrat committed:
Updated the logic for create packet setLength() function was added #AB73

909Ahmed committed:
finally fixed the checkstyles.

alonot committed:
Merge pull request #17 from alonot/priyanshu/integration/networking

Priyanshu/integration/networking

alonot committed:
Refractored code and Codec serializer changed from String to byte[]

loganvegeta committed:
Merge pull request #33 from loganvegeta/vishwaa/queuemanager

Updated the logic for create packet setLength() function was added #AB73

loganvegeta committed:
Merge pull request #34 from loganvegeta/asad/checkStyleFixes1

finally fixed the checkstyles.

LOGANATH07 committed:
git added config/checkstyle

Chirag9528 committed:
Merge pull request #19 from alonot/priyanshu/integration/networking

Refractored code and Codec serializer changed from String to byte[]

BhupathiVarun committed:
removed TestVideo.java  from Capture folder

BhupathiVarun committed:
modified VideoCapture.java

andhkr committed:
Initial Structure and PipeLine created

BhupathiVarun committed:
removed warnings, removed FrameCapture

BhupathiVarun committed:
changed version in pom.xml for java to 24

alonot committed:
Merge pull request #18 from alonot/Video_CAP

Video cap

Shubham04567 committed:
Test written for p2p2client AB#98

Chirag9528 committed:
VideoComponents and CaptureComponents refractored
> Rescale bug fix in PacketGenerator

Shubham04567 committed:
Merge branch 'module/networking' of github.com:loganvegeta/Java-Core-Communicator-SWE into p2pclient-implementation

# Conflicts:
#	java/module-networking/src/main/java/com/swe/networking/P2PClient.java

sandeepkumar009 committed:
BackgroundCaptureManager added

Chirag9528 committed:
Fix: BackgroundCapture added

andhkr committed:
Inverse DCT implemented

Chirag9528 committed:
CheckStyle Fix

andhkr committed:
Merge pull request #20 from alonot/sandeep-chirag/refractor-fix

VideoComponents and CaptureComponents refractored

loganvegeta committed:
Merge pull request #35 from loganvegeta/p2pclient-implementation

Test written for p2p2client AB#98

LOGANATH07 committed:
checkstyle fixes in p2pClient

Devansh-Kesan committed:
Implemented ZigZag and RLE Encoder for JpegDecoder

Chirag9528 committed:
fix: Refactor BilinearScaler.java

Refactored and optimized bilinearScaler.java file.

andhkr committed:
fix some checkstyle error

andhkr committed:
Merge branch 'anup/jpegCompressor' of github.com:alonot/Java-Core-Communicator-SWE into anup/jpegCompressor

# Conflicts:
#	java/module-screen-video/src/main/java/com/swe/ScreenNVideo/Codec/Compressor.java

alonot committed:
Merge branch 'anup/jpegCompressor' of github.com:alonot/Java-Core-Communicator-SWE into optimization

# Conflicts:
#	.gitignore
#	java/module-screen-video/src/main/java/com/swe/ScreenNVideo/Codec/JpegCodec.java

alonot committed:
encodeDecodeRLE bug fix and test

alonot committed:
working with Networking module

udithdoddapaneni committed:
chunkManager updated per requirements AB#103

loganvegeta committed:
Merge pull request #36 from loganvegeta/udith/networking/chunkManager3

chunkManager updated per requirements AB#103

LOGANATH07 committed:
Implemented basic Netwroking module AB#112

LOGANATH07 committed:
Added chunk in Main server AB#112

andhkr committed:
Fix: Refractor and bug fix
> Used HuffMan Encoding
> Resultant array size
> CheckStyle Fix

alonot committed:
Merge branch 'anup/jpegCompressor' of github.com:alonot/Java-Core-Communicator-SWE into optimization

# Conflicts:
#	java/module-screen-video/src/main/java/com/swe/ScreenNVideo/Codec/JpegCodec.java
#	java/module-screen-video/src/main/java/com/swe/ScreenNVideo/MediaCaptureManager.java

Devansh-Kesan committed:
HuffMan Encoding Initial version.
To be optimized

alonot committed:
Divided PacketGenerator into 2 threads
1. Some Logging added

Shubham04567 committed:
send method in p2pclient fixed and module network type added AB#98

loganvegeta committed:
Merge pull request #37 from loganvegeta/p2pclient-implementation

send method in p2pclient fixed and module network type added AB#98

andhkr committed:
fixing offset scaling

alonot committed:
Merge branch 'anup/jpegCompressor' of github.com:alonot/Java-Core-Communicator-SWE into optimization

andhkr committed:
Tests and bug fixes

andhkr committed:
Codec working

andhkr committed:
HuffMan Test added.

andhkr committed:
CheckStyle fix in Codec/

andhkr committed:
Merge branch 'module/ScreenNVideo' of github.com:alonot/Java-Core-Communicator-SWE into anup/jpegCompressor

# Conflicts:
#	.gitignore
#	java/module-screen-video/src/main/java/com/swe/ScreenNVideo/Codec/JpegCodec.java

alonot committed:
Merge pull request #22 from alonot/anup/jpegCompressor

New JPEG Compressor #AB59

909Ahmed committed:
broadcast flag honoured + P2PCluster implements P2PUser + Module connectionType

Vishwaa-Idiosyncrat committed:
Updated logic for concurrent access and packet loss, isEmpty function has been added #AB73

loganvegeta committed:
Merge pull request #38 from loganvegeta/vishwaa/queuemanager

Updated logic for concurrent access and packet loss, isEmpty function…

loganvegeta committed:
Merge pull request #39 from loganvegeta/asad/BroadCast

broadcast flag honoured + P2PCluster implements P2PUser + Module conn…

alonot committed:
Merge branch 'module/ScreenNVideo' of github.com:alonot/Java-Core-Communicator-SWE into optimization

LOGANATH07 committed:
Chunk manager added to simple  networking

LOGANATH07 committed:
Merge branch 'module/networking' of github.com:loganvegeta/Java-Core-Communicator-SWE into module/networking

alonot committed:
Merge branch 'module/networking' of github.com:loganvegeta/Java-Core-Communicator-SWE into optimization

alonot committed:
Updated CPackets with feedNumber and isFullImage info
along with height and width

alonot committed:
Moved ScreenShot in Codec to encode 's argument

alonot committed:
Few bug fixes and switching back to no Compression

alonot committed:
Merge pull request #23 from alonot/optimization

Optimization and Image Synchronization
